backend/codesirius/internal_api/views/apikey.py

```
@@ -0,0 +1,53 @@
+            api_key = serializer.save(created_by=request.user)
+            return CodesiriusAPIResponse(
+                data=APIKeySerializer(api_key).data,
+                status_code=status.HTTP_201_CREATED,
+                message="API key created",
+            )
+
+
+class APIKeyDestroyAPIView(APIView):
+    permission_classes = [IsStaff]
+
+    def delete(self, request: Request, api_key_id: int) -> CodesiriusAPIResponse:
```

Ruff (0.8.2)

Unused method argument: `request`

(ARG002)